inference-engine/include/ie_precision.hpp

```diff
@@ -18,411 +18,419 @@
 namespace InferenceEngine {
 
 /**
- * @brief This class holds precision value and provides precision related operations
+ * @brief This class holds precision value and provides precision related
+ * operations
  */
 class Precision {
 public:
-    /** Enum to specify of different  */
-    enum ePrecision : uint8_t {
-        UNSPECIFIED = 255, /**< Unspecified value. Used by default */
-        MIXED = 0,         /**< Mixed value. Can be received from network. No applicable for tensors */
-        FP32 = 10,         /**< 32bit floating point value */
-        FP16 = 11,         /**< 16bit floating point value, 5 bit for exponent, 10 bit for mantisa */
-        BF16 = 12,         /**< 16bit floating point value, 8 bit for exponent, 7 bit for mantisa*/
-        Q78 = 20,          /**< 16bit specific signed fixed point precision */
-        I16 = 30,          /**< 16bit signed integer value */
-        U8 = 40,           /**< 8bit unsigned integer value */
-        I8 = 50,           /**< 8bit signed integer value */
-        U16 = 60,          /**< 16bit unsigned integer value */
-        I32 = 70,          /**< 32bit signed integer value */
-        U32 = 74,          /**< 32bit unsigned integer value */
-        I64 = 72,          /**< 64bit signed integer value */
-        U64 = 73,          /**< 64bit unsigned integer value */
-        BIN = 71,          /**< 1bit integer value */
-        BOOL = 41,         /**< 8bit bool type */
-        CUSTOM = 80        /**< custom precision has it's own name and size of elements */
-    };
+  /** Enum to specify of different  */
+  enum ePrecision : uint8_t {
+    UNSPECIFIED = 255, /**< Unspecified value. Used by default */
+    MIXED = 0, /**< Mixed value. Can be received from network. No applicable for
+                  tensors */
+    FP32 = 10, /**< 32bit floating point value */
+    FP16 = 11, /**< 16bit floating point value, 5 bit for exponent, 10 bit for
+                  mantisa */
+    BF16 = 12, /**< 16bit floating point value, 8 bit for exponent, 7 bit for
+                  mantisa*/
+    FP64 = 13, /**< 64bit floating point value */
+    Q78 = 20,  /**< 16bit specific signed fixed point precision */
+    I16 = 30,  /**< 16bit signed integer value */
+    U8 = 40,   /**< 8bit unsigned integer value */
+    I8 = 50,   /**< 8bit signed integer value */
+    U16 = 60,  /**< 16bit unsigned integer value */
+    I32 = 70,  /**< 32bit signed integer value */
+    U32 = 74,  /**< 32bit unsigned integer value */
+    I64 = 72,  /**< 64bit signed integer value */
+    U64 = 73,  /**< 64bit unsigned integer value */
+    BIN = 71,  /**< 1bit integer value */
+    BOOL = 41, /**< 8bit bool type */
+    CUSTOM = 80 /**< custom precision has it's own name and size of elements */
+  };
 
 private:
-    struct PrecisionInfo {
-        /** @brief Size of underlined element */
-        size_t bitsSize = 0;
+  struct PrecisionInfo {
```

Comment:
Why did you change indentation?

Reply:
I used default code formatter that probably messed up the file. I'm reverting it and adding tests.
@tsocha